app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -209,6 +215,14 @@ val MIGRATION_6_7 = object : Migration(6, 7) {
     }
 }
 
+val MIGRATION_7_8 = object : Migration(7, 8) {
```

Comment:
Really surprised the IDE isn't telling you that this is unused.

Reply:
I had diff view open, and didn't realize it wouldn't tell me that when diff mode was open.